app/bundles/CoreBundle/Views/Default/head.html.twig

```diff
@@ -1,6 +1,12 @@
 <head>
     <meta charset="UTF-8" />
-    <title>{{ headerTitle|striptags }} | {{ pageTitle|default('Mautic')|striptags }}</title>
+    <title>
+        {% if headerTitle is defined and headerTitle is not empty %}
+            {{ headerTitle|replace({'<': ' <'})|striptags }}
+        {% else %}
+            {{ pageTitle|default('Mautic')|striptags }}
+        {% endif %}
+    </title>
```

Comment:
This changes the logic, I'm not sure this is what we want. It might be better to write it like `{{ headerTitle|default('')|replace({'<': ' <'})|striptags }} | {{ pageTitle|default('Mautic')|striptags }}`. I'm also curious why the use case of adding a space in front of `<`. It should find all the tags regardless of if there is a space there or not

Reply:
I don't know about the replace of '<'. Didn't make much sense to me either but it's in the original php so I put it in.
Does this logic really change functionality that much? I'll take a closer look at that output, I figured my code essentially did the same.

Reply:
I see, I wasn't looking at the php template. The twig equivalent of the php would `{{ headerTitle|default('')|replace({'<': ' <'})|striptags }} | {{ pageTitle|default('Mautic') }}`. I would skip the `if` and there in the php template, page title is always displayed.